corehq/apps/cleanup/deletable_doc_types.py

```diff
@@ -23,5 +23,6 @@
     'ForwardingRule': (MAIN_DB,),
     'ForwardingRule-Deleted': (MAIN_DB,),
     'GlobalAppConfig': (settings.NEW_APPS_DB,),
+    'RegistrationRequest': (MAIN_DB,),
```

Comment:
perhaps done later but we should also update the domain deletion command to remove the SQL data

Reply:
52983602ba6 (in https://github.com/dimagi/commcare-hq/pull/26556)